digitalpublicgoods/notesmaster.json

```diff
@@ -0,0 +1,121 @@
+    },
+    "dataPrivacySecurity": {
+      "collectsPII": "Yes",
+      "typesOfDataCollected": [
+        "First Name",
+        "Surname",
+        "Email address",
+        "School or Institution of Learning",
+        "Country of teaching or learning"
+      ],
+      "thirdPartyDataSharing": "No",
+      "dataSharingCircumstances": [
+        ""
+      ],
+      "ensurePrivacySecurity": "Yes",
+      "privacySecurityDescription": "The project only requires a name and an email address or username if the person does not have an email. There is no validation of email required. No collection of date of birth, gender, or information on geographical location. This reduces any barriers for potential users to get started."
```

Comment:
Please provide additional details on how PII data is handled and stored to ensure the privacy and security of this data and that it has taken steps to prevent adverse impacts resulting from its collection, storage and distribution.

Reply:
Below is the response provided by the `NotesMaster` team on how they handle data privacy and security of PII data:

`Security of your personal information: We will take reasonable technical and organisational precautions to prevent the loss, misuse or alteration of your personal information. \nYou acknowledge that the transmission of information over the Internet is inherently insecure, and we cannot guarantee the security of data sent over the Internet.\n You are responsible for keeping your password and other login details confidential. We will not ask you for your password (except when you log in to the website).`